.github/workflows/test.yml

```
@@ -127,11 +130,17 @@ jobs:
           done
           grep -h -v "^mode:" *.coverage >> coverage.txt
           rm -f *.coverage
+      # Sets CODECOV_TOKEN as an environment variable for the next step
+      - name: Get Codecov token
+        uses: grafana/shared-workflows/actions/get-vault-secrets@main
```

```suggestion
        uses: grafana/shared-workflows/actions/get-vault-secrets@get-vault-secrets/v1.2.1
```
This is the recommended form